net/netlink/netlink_notifier.c

```diff
@@ -109,6 +109,16 @@ int netlink_notifier_teardown(FAR struct netlink_conn_s *conn)
   if (conn->key > 0)
     {
       ret = work_notifier_teardown(conn->key);
+
+      if (ret == -ENOENT)
```

Comment:
should we just drop the return value? there are other place check the return value which may suffer the same problem. So it's better to change the return type to void and fix all place.

Reply:
Thanks!  Agreed.